ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/MacOSX.yml

```
@@ -96,7 +96,7 @@
   homebrew: "name={{ item }} state=present"
   with_items: "{{ Build_Tool_Packages_NOT_10_12 }}"
   when:
-    - not macos_version | regex_search("10.12")
+    - not macos_version.stdout | regex_search("10.12")
```

```suggestion
    - not (macos_version.stdout | regex_search("10.12"))
```
This reads better to me, personally :-)

---

Just for you 😉